homeassistant/components/sensor/fitbit.py

```diff
@@ -238,8 +238,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         for resource in config.get("monitored_resources",
                                    FITBIT_DEFAULT_RESOURCE_LIST):
             dev.append(FitbitSensor(authd_client, config_path, resource,
-                                    hass.config.temperature_unit ==
-                                    TEMP_CELSIUS))
+                                    hass.config.units.name == SYSTEM_METRIC))
```

Comment:
Since this is such a common check, why not have a boolean on  the unit system `is_metric`: `hass.config.units.is_metric`